components/slider/stories/template.js

```diff
@@ -238,3 +242,40 @@ export const Template = ({
 			`)}
 		</div>`;
 };
+
+export const SizingTemplate = (args, context) => 
```

Comment:
Can you update this to use the `avatar.stories.js` `SizeOptions` story as a template for how to use the Sizes function from the utility decorators instead of recreating it?